### PR TITLE
2372 in ssdc fix queue names now to be consistent

### PR DESCRIPTION
**Pull request by LukeLoze1**
queue renaming

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ WORKDIR /home/toolbox
 ENV RABBITMQ_SERVICE_HOST rabbitmq
 ENV RABBITMQ_SERVICE_PORT 5672
 ENV RABBITMQ_VHOST /
-ENV RABBITMQ_QUEUE case.sample.inbound
+ENV RABBITMQ_QUEUE SampleLoader.Case.Sample
```
Comment (NickGrantONS) [inline Dockerfile:16]: case looks wrong

Comment (NickGrantONS) [inline Dockerfile:16]:
```suggestion
ENV RABBITMQ_QUEUE supportTool.caseProcessor.sample
```
```suggestion
ENV RABBITMQ_QUEUE SampleLoader.Case.Sample
```

Comment (NickGrantONS) [inline Dockerfile:16]:
```suggestion
ENV RABBITMQ_QUEUE supportTool.caseProcessor.sample
```
```suggestion
ENV RABBITMQ_QUEUE SampleLoader.Case.Sample
```

Comment (NickGrantONS) [inline Dockerfile:16]:
```suggestion
ENV RABBITMQ_QUEUE supportTool.caseProcessor.sample
```
```suggestion
ENV RABBITMQ_QUEUE SampleLoader.Case.Sample
```

File: sample_loader/rabbit_context.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ def __init__(self, **kwargs):
         self._exchange = kwargs.get('exchange') or os.getenv('RABBITMQ_EXCHANGE', '')
         self._user = kwargs.get('user') or os.getenv('RABBITMQ_USER', 'guest')
         self._password = kwargs.get('password') or os.getenv('RABBITMQ_PASSWORD', 'guest')
-        self.queue_name = kwargs.get('queue_name') or os.getenv('RABBITMQ_QUEUE', 'case.sample.inbound')
+        self.queue_name = kwargs.get('queue_name') or os.getenv('RABBITMQ_QUEUE', 'SampleLoader.Case.Sample')
```
Comment (NickGrantONS) [inline sample_loader/rabbit_context.py:15]:
```suggestion
        self.queue_name = kwargs.get('queue_name') or os.getenv('RABBITMQ_QUEUE', 'supportTool.caseProcessor.sample')
```
```suggestion
        self.queue_name = kwargs.get('queue_name') or os.getenv('RABBITMQ_QUEUE', 'SampleLoader.Case.Sample')
```

**Review by NickGrantONS — CHANGES_REQUESTED** [review]
Qwrong.com

**Review by NickGrantONS — APPROVED** [review]
Builds and ATs passing 👍

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**